inbox/accounts.go

```diff
@@ -2,10 +2,13 @@ package inbox
 
 import (
 	"context"
+	"encoding/json"
 	"fmt"
-	"github.com/pkg/errors"
+	"io"
 	"net/http"
 
+	"github.com/pkg/errors"
+
 	openapi "go.opscenter.dev/james-go-client"
```

Comment:
Just name it as `apis`. Name `Openapi` is conflicting and it is also a open source specification